lib/Doctrine/bootstrap_doctrine_TEMPLATE.php

```diff
@@ -82,7 +80,10 @@
     /////////////////////////////////////////////////////////////////////////////////////////////
 
 
+    $evm = new EventManager();
 
+    // ** For Oracle installations uncomment the following line:
```

Comment:
Is this referenced in the documentation? I can imagine this could easily be missed.

Reply:
it has now been moved into the ORACLE specific block to make it less likely to be missed.